plugins/woocommerce/i18n/states.php

```diff
@@ -409,6 +409,32 @@
 		'DZ-48' => __( 'Relizane', 'woocommerce' ),
 	),
 	'EE' => array(),
+	'EC' => array( // Ecuador states.
+		'EC-A' => __( 'Azuay', 'woocommerce' ),
+		'EC-B' => __( 'Bolívar', 'woocommerce' ),
+		'EC-F' => __( 'Cañar', 'woocommerce' ),
+		'EC-C' => __( 'Carchi', 'woocommerce' ),
+		'EC-H' => __( 'Chimborazo', 'woocommerce' ),
+		'EC-X' => __( 'Cotopaxi', 'woocommerce' ),
+		'EC-O' => __( 'El Oro', 'woocommerce' ),
+		'EC-E' => __( 'Esmeraldas', 'woocommerce' ),
+		'EC-W' => __( 'Galápagos', 'woocommerce' ),
+		'EC-G' => __( 'Guayas', 'woocommerce' ),
+		'EC-I' => __( 'Imbabura', 'woocommerce' ),
+		'EC-L' => __( 'Loja', 'woocommerce' ),
+		'EC-R' => __( 'Los Ríos', 'woocommerce' ),
+		'EC-M' => __( 'Manabí', 'woocommerce' ),
+		'EC-S' => __( 'Morona Santiago', 'woocommerce' ),
```

Comment:
Please hyphenate this as to conform to the Unicode CLDR standard. https://github.com/unicode-org/cldr/blob/release-40/common/subdivisions/en.xml#L1243